apps/root/src/frame/components/background-grid/light.tsx

```diff
@@ -0,0 +1,753 @@
+import React from 'react';
+import { CustomSvgIcon, SvgIconProps } from 'ui-library';
+
+interface IconProps extends SvgIconProps {
+  size?: string;
+}
+
+export default function LightBackgroundGrid({ size, ...props }: IconProps) {
+  return (
+    <CustomSvgIcon viewBox="0 0 1921 668" {...props} sx={{ width: '100%', height: '100%' }} fill="none">
```

Comment:
I believe we should split the Grid from the gradient in different approaches. The grid is well positioned, but the gradient should continue bellow the 100vh
<img width="1255" alt="Screenshot 2024-03-06 at 14 35 54" src="https://github.com/Mean-Finance/dca-fe/assets/144699868/e2524327-fd0c-470a-941c-98f7beb3b2eb">

I did something similar in the Swap landing!

Reply:
Oh shit forgot to set it as fixed

Reply:
It's not reaching the bottom yet 😢 
<img width="946" alt="Screenshot 2024-03-06 at 15 27 07" src="https://github.com/Mean-Finance/dca-fe/assets/144699868/f63fc407-904f-4f0f-b8b2-3dc1fb03de30">